htmlreport/setup.py

```diff
@@ -1,21 +1,31 @@
 #!/usr/bin/env python3
+import shutil
+from setuptools import find_packages, setup
 
-from setuptools import setup
-
-with open('README.txt') as f:
+with open("README.txt") as f:
     readme = f.read()
 
+shutil.copy("cppcheck-htmlreport", "cppcheck_htmlreport/run.py")
+
 setup(
-    name="cppcheck",
-    description='Python script to parse the XML (version 2) output of ' +
-                'cppcheck and generate an HTML report using Pygments for syntax ' +
-                'highlighting.',
+    name="cppcheck-htmlreport",
+    description=(
+        "Python script to parse the XML (version 2) output of "
+        "cppcheck and generate an HTML report using Pygments for syntax "
+        "highlighting."
+    ),
     long_description=readme,
-    author='Henrik Nilsson',
-    url='https://github.com/danmar/cppcheck',
-    license='GPL',
-    scripts=[
-        "cppcheck-htmlreport",
-    ],
-    install_requires=['Pygments']
+    author="Henrik Nilsson",
+    url="https://github.com/danmar/cppcheck",
+    license="GPL",
+    packages=find_packages(exclude=("tests", "docs")),
+    use_scm_version={"root": ".."},
+    entry_points={
+        "console_scripts": [
+            "cppcheck-htmlreport = cppcheck_htmlreport:run.main",
+        ]
+    },
+    test_suite="tests",
+    install_requires=["Pygments"],
+    setup_requires=["setuptools>=60", "setuptools-scm>=7.0"],
```

Comment:
Why do we need those specific versions? Please add a small comment.

Reply:
Sure, will add a comment. That is the requirement of the stable version of the [`setuptool-scm`](https://setuptools-scm.readthedocs.io/en/stable/) package that allows getting package version from a source control.

Reply:
Comment was added.